src/components/LikedList.js

```diff
@@ -1,14 +1,52 @@
-import React from "react";
+import axios from "axios";
+import React, {useEffect, useState} from "react";
 import {useVideo} from "../context/video-context";
 import {VideoList} from "./VideoList";
-
-export function LikedList() {
+import "./LikedList.css";
+export const LikedList = () => {
   const {
     value: {likes},
+    dispatch,
   } = useVideo();
+  const [error, setError] = useState(null);
+  useEffect(() => {
+    const anonFunc = async () => {
+      try {
+        const response = await axios.get(
+          `https://videolib.amansethi00.repl.co/likedVideos`,
+          {
+            headers: {
+              Authorization: `${localStorage?.getItem(
+                "username"
+              )}:${localStorage?.getItem("password")}`,
+            },
+          }
+        );
+        if (response.data.success) {
+          console.log(response.data);
+          dispatch({
+            type: "SET_LIKEDVIDEOS",
+            payload: response.data,
+          });
+        }
+      } catch (error) {
+        console.log({error});
+        setError(error.response.data.message);
+      }
+    };
+    anonFunc();
+  }, []);
   return (
-    <div>
-      <VideoList value={likes} title={"Liked Videos"} />
-    </div>
+    <>
+      {error !== null && (
+        <div className="toast-error mg-bottom-1 errorBox">
```

Comment:
The error message should disappear on it's own after a certain interval in my opinion. That way user doesn't have to click several times just to make a modal disappear. It's not a good UX. Try using setTimeOut or any other method you prefer. Also, it would be much better if instead of an error message, you replace the "like button" with a "remove from like" button. I know this isn't a finalized version, but these are some UI stuff I've noticed so far.